md/bonita-studio-download-installation.md

```diff
@@ -8,9 +8,9 @@ If you face any difficulties in this getting started tutorial, please [ask for h
 
 ## Download Bonita Studio
 
-To download the latest version of Bonita Studio, open the [download page](https://www.bonitasoft.com/downloads) and click on the "Download" button. This will start the download of Bonita Studio installer for your operating system.
+To download the latest version of Bonita Studio, open the [download page](https://www.bonitasoft.com/downloads) and click on the **Download** button. This will start the download of Bonita Studio installer for your operating system.
 
-When the download is finished, you should have one of the following files on your computer (x.y.z refers to the version of Bonita Studio, e.g. 7.9.2):
+When the download is finished, you should have one of the following files on your computer (x.y.z refers to the version of Bonita Studio, e.g. 7.10.0):
```

Comment:
I suggest to use
```suggestion
When the download is finished, you should have one of the following files on your computer (x.y.z refers to the version of Bonita Studio, e.g. ${varVersion}.0):
```
so that it replaces in the example with the current version of the documentation consulted